classes/class-wc-connect-api-client.php

```diff
@@ -295,6 +295,30 @@ public function auth_test() {
 			return $this->request( 'GET', '/connection/test' );
 		}
 
+		/** Heartbet test
+		 * 
+		 * @return true|WP_Error
+		 */
+		public function is_alive() {
+			return $this->request( 'GET', '' );
+		}
+
+		/** Heartbet test with transient cache.
+		 * 
+		 * @return true|WP_Error
+		 */
+		public function is_alive_cached() {
+			$connect_server_is_alive_transient = get_transient( 'connect_server_is_alive_transient' );
+			if ( false !== $connect_server_is_alive_transient ) {
+				return true;
+			}
+
+			$is_alive_request = $this->is_alive();
+			$new_is_alive = ! is_wp_error( $is_alive_request );
+			set_transient( 'connect_server_is_alive_transient', $new_is_alive, MINUTE_IN_SECONDS );
```

Comment:
Wow, get_transient is annoying with booleans. My suggestion didn't fully fix it.  `true` is resulting in `"1"` and `false` is resulting in `""`. The following makes it work:
```suggestion
			if ( $new_is_alive ) {
				set_transient( 'connect_server_is_alive_transient', true, MINUTE_IN_SECONDS );
			}
```